### PR TITLE
implement ATM cash-in quest

### PR DESCRIPTION
**Pull request by mnalis**
Closes: https://github.com/streetcomplete/StreetComplete/issues/4292
([tested](https://github.com/mnalis/StreetComplete/actions/runs/2976059075) and seems to work ok)

Note: I re-used _"ATM Operator"_ quest `ic_quest_money` icon ("wad of notes" image)
Maybe it is fine as-is (for similar quests to have same icon), or a modified icon should be created.

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -605,6 +605,7 @@ Before uploading your changes, the app checks with a &lt;a href=\"https://www.we
 
     <string name="quest_airConditioning_title">"Is this place air-conditioned?"</string>
 
+    <string name="quest_atm_cashin_title">"Does this ATM allows depositing cash?"</string>
```
Comment (andrewharvey) [inline app/src/main/res/values/strings.xml:608]:
```suggestion
    <string name="quest_atm_cashin_title">"Does this ATM allow depositing cash?"</string>
```

Comment (andrewharvey) [reply app/src/main/res/values/strings.xml:608]: Or you could say.

> Can you deposit cash at this ATM?

Comment (mnalis) [reply app/src/main/res/values/strings.xml]: thanks @andrewharvey , added that as it sounds even better!

**Comment by matkoniecz** [comment]
>([tested](https://github.com/mnalis/StreetComplete/actions/runs/2976059075) and seems to work ok)

just that it builds or also in use?

**Comment by mnalis** [comment]
In use too, e.g. https://www.openstreetmap.org/node/5181271721/history
It adds the `cash_in=yes` (or `no`) tag, and then the quest is not shown again (as it shouldn't be).

**Review by westnordost — CHANGES_REQUESTED** [review]
Could you use camel case "CashIn"?

File: app/src/main/java/de/westnordost/streetcomplete/quests/atm_cashin/AddAtmCashin.kt [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+import de.westnordost.streetcomplete.quests.YesNoQuestForm
+import de.westnordost.streetcomplete.util.ktx.toYesNo
+
+class AddAtmCashin : OsmFilterQuestType<Boolean>() {
+
+    override val elementFilter = """
+        nodes with
+         amenity = atm
+         and !cash_in
+    """
```
Comment (westnordost) [inline app/src/main/java/de/westnordost/streetcomplete/quests/atm_cashin/AddAtmCashin.kt:19]:
```suggestion
    override val elementFilter = "nodes with amenity = atm and !cash_in"
```
I think this fits quite well into one line 🤷

**Comment by mnalis** [comment]
> Could you use camel case "CashIn"?

CamelCase added.

**Review by westnordost — APPROVED** [review]

**Pull request merged**